src/mame/drivers/gigatron.cpp

```diff
@@ -82,6 +83,11 @@ void gigatron_state::blinkenlights(uint8_t data)
 	lights_changed ^= light;
 }
 
+uint8_t gigatron_state::inputs()
+{
+	return m_io_inputs->read() ^ 0xFF;
+}
+
```

Comment:
You could avoid this trampoline entirely either by using active high inputs in the input port definitions, or by using something like `m_maincpu->ir_cb().set_ioport("GAMEPAD").invert();` in your machine configuration.